cmd/login-consent-server/templates/banklogin.html

```diff
@@ -23,9 +23,10 @@
 
     <style>
         .gradient {
-            background:linear-gradient(-180deg, #8631a0 0%, #360b4c 100%);
+            background: linear-gradient(-180deg, #8631a0 0%, #360b4c 100%);
```

Comment:
You can actually drop `-180deg`, it is not needed. Just double check it matched the designs (not the auto generated code in abstract)